win_etw_macros/src/tests.rs

```diff
@@ -51,6 +48,19 @@ impl syn::parse::Parse for CompileErrors {
 
 fn test_worker(attrs: TokenStream, input: TokenStream, expected_errors: &[&'static str]) {
     let output = trace_logging_events_core(attrs, input);
+    if std::env::var("WIN_ETW_SHOW_OUTPUT").is_ok() {
```

Comment:
What is this for? Should we document this env var?

Reply:
It's just so that I can see the macro output from unit tests, during development.

Reply:
Added a comment describing this env var.